mmv1/third_party/terraform/services/compute/resource_compute_instance.go.erb

```diff
@@ -3020,3 +3025,43 @@ func serviceAccountDiffSuppress(k, old, new string, d *schema.ResourceData) bool
 	}
 	return true
 }
+
+// isEmptyServiceAccountBlock is used to work around an issue when updating
```

Comment:
Apologies for the late review, I'm not sure how this slipped through. Only question I have is what's the reason for commenting this whole block of code? What's stopping us from removing it entirely?

Reply:
Hey, there is no code commented, this is just a (very) long comment before the function. The behavior around "scopes" is not obvious and I wanted to explain this in the comment.